docs/example-react-transition-group.mdx

```diff
@@ -95,7 +95,7 @@ test('you can mock things with jest.mock', () => {
   )
   fireEvent.click(getByText(/toggle/i))
   expect(CSSTransition).toHaveBeenCalledWith(
-    {in: true, ...defaultProps},
+    {in: false, ...defaultProps},
```

Comment:
Thanks for fixing this snippet, could you also import add `useState` to the imports please?

Reply:
Sorry, I missed the notification